BUILDING_FROM_SOURCE.md

```diff
@@ -202,26 +202,14 @@ export CPPFLAGS="-I/usr/local/opt/llvm/include"
 **Warning** While `cargo build` works on windows, linking roc programs does not yet, see issue #2608. This also means the repl, the editor and many tests will not work on windows.
 The official LLVM pre-built binaries for Windows lack features that roc needs. Instead:
 
-1. Download and install [Build Tools for Visual Studio 2019](https://visualstudio.microsoft.com/thank-you-downloading-visual-studio/?sku=BuildTools&rel=16) (a full Visual Studio install should work too; the Build Tools are just the CLI tools, which is all we need)
```

Comment:
On the github hosted server I did not actually need this, it might be that this step is unnecessary or the github server came pre-installed with what we needed. I'd like to try leaving out and see if it breaks anything on clean windows installs.

Reply:
for me installing rustup (for cargo) prompted me to install some official msvc tooling. That was enough for me and I did not need this special thing